### ISSUE TITLE
Suggestion: Villagers leave the last item in a chest

### ISSUE THREAD

**Issue by JohnyDL** [TangoTek/TekTopia-Community#382]
**Describe the Feature**
Villagers leave the last item in storage chests

**Why do you think this improvement will help the mod?**
The villagers will keep things organised but only if there's an item in the chests, if you want to keep say iron ore in a specific chest and the blacksmiths take it all there's very little you can do about it. This would greatly help keep things organised for players with OCD

**Comment by LadyGwynn** [comment]
I agree with this very much. If villagers were unable to remove the last item in a stack from chests in storage, it would make it much easier to organize and keep track of what resources a busy village has available.
